web/components/dropdowns/module.tsx

```diff
@@ -77,12 +77,16 @@ const ButtonContent: React.FC<ButtonContentProps> = (props) => {
     return (
       <>
         {showCount ? (
-          <>
+          <div className="relative flex items-center gap-1">
             {!hideIcon && <DiceIcon className="h-3 w-3 flex-shrink-0" />}
-            <span className="flex-grow truncate text-left">
-              {value.length > 0 ? `${value.length} Module${value.length === 1 ? "" : "s"}` : placeholder}
-            </span>
-          </>
+            <div className="flex-grow truncate max-w-40">
```

Comment:
The `max-w-40` should be accepted as a prop, as we may need different widths in different scenarios.

Reply:
To maintain consistency across all dropdowns, we adhere to the `max-w-40` styling. Therefore, passing it as a prop is not feasible.